client/scss/components/forms/_switch.scss

```diff
@@ -1,12 +1,11 @@
 @use 'sass:math';
 
-$switch-width: 40px;
-$switch-height: 20px;
+$switch-width: 36px;
+$switch-height: 18px;
 $switch-border: 2px;
 $switch-outline: 3px;
```

Comment:
should we aim to make checkboxes & switches the same height?

Reply:
Spoke with Thibaud - there is some nuance here as we have to style the actual DOM input elements, not wrappers plus they do not really get used in forms alongside checkboxes/radios.